docs/hud_data_model.md

```diff
@@ -0,0 +1,100 @@
+# HMIS HUD CSV data model
+
+Based on:
+HMIS CSV FORMAT Specifications FY2024
+VERSION 1.4
+
+## Key Relationships
+
+* One Organization can have many Projects
+* One Project can have many Enrollments
+* One Client can have many Enrollments
+* Each Enrollment is tied to exactly one Project and one Client
+* Each Enrollment can have multiple associated services, assessments, and other records
+
+## ERD
+
+Note "CoC" is not a defined table; is included in the diagram as an "implied" entity
+
+```mermaid
+erDiagram
+    CoC ||..o{ Organization : "operates in"
+    CoC ||..o{ ProjectCoC : "contains"
+    Export ||--o{ Organization : contains
+    Organization ||--o{ Project : operates
+    Project ||--o{ ProjectCoC : "has locations in"
+    Project ||--o{ Funder : "funded by"
+    Project ||--o{ Inventory : "has"
+    Project ||--o{ Affiliation : "may be affiliated with"
```

Comment:
The affiliation should always link back to another project (eg project X is affiliated with project Y). But I don't think that is super important.

Reply:
added the back reference